app/src/main/java/org/techtown/twosomeheart/presentation/detail/DetailRoute.kt

```diff
@@ -0,0 +1,178 @@
+    modifier: Modifier = Modifier
+){
+    Column(
+        modifier = modifier
+            .padding(paddingValues)
+            .background(White)
+            .fillMaxSize()
+            .verticalScroll(rememberScrollState())
+    ){
+        Topbar(
+            leadingIcon = {
+                Icon(
+                    imageVector = ImageVector.vectorResource(R.drawable.ic_back),
+                    contentDescription = stringResource(R.string.top_bar_back),
+                    modifier = Modifier.noRippleClickable(onClick = navigateUp)
+                )
+            },
+            leadingIcon2 = {
+                Icon(
+                    imageVector = ImageVector.vectorResource(R.drawable.ic_home),
+                    contentDescription = stringResource(R.string.top_bar_home),
+                )
+            },
+            text = stringResource(R.string.menu_detail_top_bar),
+            trailingIcon3 = {
+                Icon(
+                    imageVector = ImageVector.vectorResource(R.drawable.ic_basket),
+                    contentDescription = stringResource(R.string.top_bar_basket),
+                    modifier = Modifier.noRippleClickable(onClick = navigateUp),
+                    tint = Color.Unspecified
+                )
+            }
+        )
+
+        when (state) {
+            is UiState.Loading -> {}
+
+            is UiState.Empty -> {}
+
+            is UiState.Failure -> {}
+
+            is UiState.Success -> {
+                Image(
+                    painter = painterResource(R.drawable.img_detail_banana_latte),
+                    contentDescription = stringResource(R.string.menu_detail_image),
+                    modifier = Modifier.fillMaxWidth(),
+                    contentScale = ContentScale.Crop
+                )
+
+                Spacer(modifier = Modifier.height(26.dp))
+
+                MenuDetailContent(
+                    // TODO: detailmodel best menu 처리 어떻게 할건지 생각
+                    isBestMenu = true,
+                    menuName = state.data.menuName,
+                    menuDescription = state.data.menuDescription,
+                    menuPrice = state.data.menuPrice,
+                )
+
+                HorizontalDivider(
+                    thickness = 4.dp,
+                    color = Gray20
+                )
+
+                MenuNutritionColumn(
+                    nutritionText = state.data.menuNutrition
+                )
+
+                if(state.data.menuAllergy != null){
+                    MenuDetailAllergyText(
+                        allergyText = state.data.menuAllergy
+                    )
+                }
+
+                Spacer(modifier = Modifier.height(58.dp))
+
+                BlackBottomButton(
+                    text = stringResource(R.string.menu_detail_order_text),
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(horizontal = 16.dp, vertical = 16.dp)
```

Comment:
상하/좌우 모두 동일한 패딩값을 가질때에는 
```suggestion
                        .padding(16.dp)
```
로 설정해주시면 됩니다  :)

Reply:
이 부분은 놓쳤네요 감사합니다 !!